### PR TITLE
Feat/361/add UI kit copy to clipboard

### LINKED ISSUE #361: feat: add Ui-Kit CopyToClipboard to BE
Replace all instances of CopyToClipboard in BE with Ui-Kit CopyToClipboard

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
- Replace CopyToClipboard component with ui-kit component for #361 

### 😤 Problem / Goals
- We have moved the CopyToClipboard component into the casper-ui-kit 

### 🤓 Solution
- Remove the old component and update it with ui-kit

### CODE REVIEW

**Review by Thistle-Strum — APPROVED** [review]

**Review by mgibson-casper — APPROVED** [review]

File: app/src/components/utility/StyledCopyToClipboard/StyledCopyToClipboard.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { CopyToClipboard, defaultTheme } from 'casper-ui-kit';
+import { useTheme } from '@emotion/react';
+
+interface StyledCopyToClipboardProps {
+  textToCopy: string;
```
Comment (DorianKwan) [inline app/src/components/utility/StyledCopyToClipboard/StyledCopyToClipboard.tsx]:
```suggestion
  readonly textToCopy: string;
```

File: app/src/components/utility/index.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 export { Coin } from './Coin';
-export { CopyToClipboard } from './CopyToClipboard';
```
Comment (DorianKwan) [inline app/src/components/utility/index.ts:2]: Can we replace this with the styled CopyToClipboard export?  And then clean up 
`import ... from 'src/components/utility/StyledCopyToClipboard';` to `import ... from 'src/components/utility';`

**Review by DorianKwan — CHANGES_REQUESTED** [review]

**Review by mgibson-casper — APPROVED** [review]
Nice work 👍

**Review by DorianKwan — APPROVED** [review]
Awesome!

**Pull request merged**